### DISCUSSION TITLE
Script friendly returned id on ticket creation

### DISCUSSION THREAD

**Discussion by pdeschen** [ankitpokhrel/jira-cli#408] [Ideas]
As it stands, executing a ticket creation task commands 

```
$>jira issues create --parent ENGNP-90 --type Task --summary "Foo" --template description.txt --assignee p.deschenes --no-input
```

returns a nicely formatted output

```
✓ Issue created
https://example.atlassian.net/browse/ENGNP-150
```

However, I'd like to be able to easily capture the ticket number in order to do further processing

```bash
$>$TICKET=$(jira issues create ...)
$>echo $TICKET
ENGNP-150
```

We could expend this use case across the board to be able to do something like

```
>$jira -pENGNP issues list  > out.txt
```

Maybe check for TTY terminal or not? Maybe a global flag (`--plain` or `--format=notty`)? Or it could be output as JSON and we could `jq` on it? This is somewhat similar to this [other discussion](https://github.com/ankitpokhrel/jira-cli/discussions/367) on `list --plain` so maybe having such flag global handling?

**Comment by shayn-orca** [comment]
JSON output (for `jq` support) would be amazing for me as well

**Reply by SleepyBrett** [reply]
every subcommand should support json output (anywhere there is --plain at the very least). I get that TUIs are 'what's hot', but for a tool like this it's often 'it's not'.
